cloud/getting-started/about-centreon-cloud.md

```diff
@@ -0,0 +1,40 @@
+---
+id: about-centreon-cloud
+title: About Centreon Cloud
+---
+
+## What is Centreon Cloud?
+
+Centreon Cloud is the Centreon monitoring platform installed in our Cloud infrastructure. For detailed product information, visit our [product description page](https://www.centreon.com/centreon-cloud/).
+
+## What is the SLA for Centreon Cloud?
+
+Centreon Cloud is committed to a high quality of service with the following guarantees:
+
+* The SLA for Centreon Cloud is 99.5%. More information in our [Terms and Conditions](https://www.centreon.com/legal/en/Cloud-Services-terms-ROW).
+
+* The RPO (Recovery Point Objective) is 24 hours.
+
+* The RTO (Recovery Time Objective) is 4 hours.
+
+## Where is my data hosted?
+
+Centreon uses [several hosting providers](../security/security.md#hosting-security), mainly in Europe. We are [AWS partners](https://www.centreon.com/partners/centreon-on-aws/).
+
+## How long is my data kept for?
+
+Monitoring data is kept for 13 months: performance data (metrics), downtimes, acknowledgements, comments, logs...
+All other data (e.g. your configuration) is kept until the end of your subscription.
+
+## What security measures does Centreon take?
+
+For a comprehensive overview of our security practices, visit our [Centreon Cloud security management page](../security/security.md).
+
+## Fair use policy
+
+To understand our limitations in terms of resource use, read our [Fair Use Policy](https://www.centreon.com/legal/en/centreon-cloud-service-fair-use-policy).
+
+## Maintenance updates and upgrades
+
+* Your central server is fully managed and updated by Centreon. No action is required on your part.
+* Your pollers are in your infrastructure: it is your responsibility to [update or upgrade them](../installation/poller-update-upgrade.md).
```

Comment:
Can you add a link to the glossary for "pollers"?
Also "it is your responsibility to install and keep them up-to-date"